src/framing/FramedDuplexConnection.h

```diff
@@ -25,6 +25,10 @@ class FramedDuplexConnection : public virtual DuplexConnection {
   void setInput(yarpl::Reference<yarpl::flowable::Subscriber<std::unique_ptr<folly::IOBuf>>>
                     framesSink) override;
 
+  bool isFramed() override {
```

Comment:
nit: checking if the input parameter "connection" of the constructor is not already a FramedDuplexConnection.